Readme.md

```diff
@@ -107,7 +107,7 @@ You can download the package from [Maven Central](https://repo1.maven.org/maven2
 
 ## Older AEM Versions (<6.5/Cloud)
 
-You can download the package from [Maven Central](https://repo1.maven.org/maven2/de/valtech/aecu/aecu.ui.apps/) or our [releases section](https://github.com/valtech/aem-easy-content-upgrade/releases). The aecu.ui.apps package will install the AECU software. It requires that you installed [Groovy Console](https://github.com/OlsonDigital/aem-groovy-console) before.
+You can download the package from [Maven Central](https://repo1.maven.org/maven2/de/valtech/aecu/aecu.ui.apps/) or our [releases section](https://github.com/valtech/aem-easy-content-upgrade/releases). The aecu.ui.apps package will install the AECU software. It requires that you installed [Groovy Console](https://github.com/orbinson/aem-groovy-console) before.
```

Comment:
This change can be reverted as for <6.5 the old Groovy Console needs to be used.

Reply:
yes, good point, reverted